core/execution/engine.go

```diff
@@ -206,6 +206,14 @@ func (e *Engine) ReloadConf(cfg Config) {
 	}
 }
 
+func (e *Engine) OnEpochEvent(ctx context.Context, epoch types.Epoch) {
+	// TODO
+}
+
+func (e *Engine) OnEpochRestore(ctx context.Context, epoch types.Epoch) {
+	// TODO
+}
+
```

Comment:
Notthin to do the for not too? is that for a followup PR?

Reply:
yes, in the spots PR this will call all markets but this is in the following spot pr